src/suites/cts/buffers/mapping_test.ts

```diff
@@ -0,0 +1,33 @@
+import { GPUTest } from '../gpu_test.js';
+
+export class MappingTest extends GPUTest {
+  checkMapWrite(buffer: GPUBuffer, arrayBuffer: ArrayBuffer, size: number): Promise<void> {
```

Comment:
nit: Looking at the function in isolation, it's hard to see that `arrayBuffer` is the mapped buffer. Maybe name it `mappedContents`?